.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -8,6 +8,192 @@ body:
         Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
 
         Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component(s)
+      description: Which component(s) does your bug report concern?
+      multiple: true
+      options:
+      # Start Collector components list
+      - cmd/configschema
```

Comment:
Is there a way during CI to ensure that this list is up to date?  We do some similar kinds of checks in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/b737b991986d8213187f0ae862e1d57da7a069ce/.github/workflows/build-and-test.yml#L121

Reply:
Good call, I added a check.